Minor tweaks and improvements

MohamadNateqi opened the pull request:
This PR applies the following changes:
1. The style has been updated to ensure the sub-tab on the settings page always displays correctly. Previously, it was only working for the debug tab.
2. The `validate()` method now accepts null values. If it doesn't accept nulls, it could raise an error when no settings are saved, such as when only unchecked checkboxes are present on the page.
3. Small improvement.

alexmigf commented:
The style of this sub-menu changed to the WP default:

![image](https://github.com/user-attachments/assets/d8a9b5e8-e519-4c65-8b72-8d8626a53884)

But we can use the previous style in all sub-menus?

MohamadNateqi commented:
> Maybe we can use the previous style in all sub-menus?

I like the idea.

alexmigf commented:
Here looks a bit odd, maybe because the page doesn't have a main title:

![image](https://github.com/user-attachments/assets/6ab62eb0-3b1c-4fa2-82fd-2f3d1d67f63a)

MohamadNateqi commented:
@alexmigf What do you think now?

![image](https://github.com/user-attachments/assets/88815e48-afc6-4bd8-976d-913bb0fadae5)

alexmigf commented:
I don't see much of a difference either, but I was referring to adding a title between the two menus. What do you think?

@n-rusu Could you share your input as well?

MohamadNateqi commented:
I updated the sub menu style to this:

![image](https://github.com/user-attachments/assets/73271231-43f8-4e72-81b2-fbde568b4ed9)

Still, I'm not sure if this is the best way to implement it. Also, I added a bottom border because it seemed like the distinction wasn't clear.

@n-rusu, can you take a look and share your input, please?

n-rusu commented:
Looks good👌
